app/controllers/contestants_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ContestantsController < ApplicationController
+  def index
+    @contestants = Contestant.all
+    @contestants_projects = ContestantProject.includes(:project).where(contestant_id: @contestants.pluck(:id))
```

Comment:
This is a cool query and it does work - couple notes: 
* I'd be interested to hear your reasoning in using `.includes` here instead of the more common `.all`...? It's not wrong, but I want to make sure you understand its use case here. 
* Using `.where` in a controller is considered an MVC infraction since it will return a more variable collection - as such, we'd want to make a model method out of it so we can test it in a model test. However, using `.pluck` here makes a lot of sense to send to the view, so your model method could use .pluck as well!